app/steps/ui/executors/number/index.js

```diff
@@ -44,13 +45,32 @@ class ExecutorsNumber extends ValidationStep {
         return [ctx.executorsNumber >= 0, 'inProgress'];
     }
 
-    nextStepOptions() {
-        const nextStepOptions = {
+    handlePost(ctx, errors, formdata, session, hostname, featureToggles) {
+        ctx.isToggleEnabled = FeatureToggle.isEnabled(featureToggles, 'screening_questions');
+
+        return [ctx, errors];
+    }
+
+    nextStepOptions(ctx) {
+        if (ctx.isToggleEnabled) {
```

Comment:
Is there a way we could assign the value for 'choice' based on the value of ctx.isToggleEnabled, like this:
const choiceValue = ctx.isToggleEnabled ? 'oneExecutorToggleOn' : 'oneExecutor';

Reply:
That way we avoid the if block and the code reuse of the options object.

Reply:
Yeah I guess we could but this code is only temporary and is getting removed in PRO-3799.

Reply:
Yep, I think thats fair enough, it's only temporary code and perhaps makes it easier to remove when we clear up the toggle